torch/csrc/jit/passes/onnx/naming.cpp

```diff
@@ -0,0 +1,80 @@
+#include <torch/csrc/jit/passes/onnx/naming.h>
+#include <torch/csrc/onnx/onnx.h>
+
+namespace torch {
+namespace jit {
+namespace onnx {
+
+namespace ONNXScopeName {
+
+using NameFunc = std::string (*)(torch::jit::ScopePtr scope);
+
+const std::string name_separator = "::";
+
+namespace {
+
+std::string nameFromRoot(
```

Comment:
Should functions follow PascalCase naming?

Reply:
I realized PyTorch uses camel case for functions everywhere else. Sorry for making the previous suggestion